addons/stock/models/stock_inventory.py

```diff
@@ -50,6 +50,7 @@ class Inventory(models.Model):
     product_ids = fields.Many2many(
         'product.product', string='Products', check_company=True,
         domain="[('type', '=', 'product'), '|', ('company_id', '=', False), ('company_id', '=', company_id)]", readonly=True,
+        context={'active_test': False},
```

Comment:
It's a bit too much. I would keep the active filter here. The matter is just to show archived product when a global inventory adjustment is ongoing